src/deluge/playback/playback_handler.cpp

```diff
@@ -2506,6 +2511,27 @@ bool PlaybackHandler::tryGlobalMIDICommands(MIDIDevice* device, int channel, int
 	return foundAnything;
 }
 
+// Returns whether the message has been used up by a note-off command
+bool PlaybackHandler::tryGlobalMIDICommandsOff(MIDIDevice* device, int channel, int note) {
+
+	bool foundAnything = false;
+
+	for (int c = 0; c < kNumGlobalMIDICommands; c++) {
```

Comment:
For now should it just check the fill command instead of all of them?

Reply:
Yeah, good point. I just used a switch statement so the next person to touch this knows the convention. I'll do a single check for now with a note to use switch for extra checks 👍

Reply:
Oh I meant the for loop - we only care about the full command so we can skip the checks to other global midi commands

Reply:
I'm not sure I understand how to go about this. Do you mean I should get rid of the for loop and then in the if statement replace [c] with the actual index of the FILL command, like [17]?

Reply:
Latest commit should be better.